### DISCUSSION TITLE
SOLVED Using CLI in Container

### DISCUSSION THREAD

**Discussion by Nollknolle** [dani-garcia/vaultwarden#3199] [General]
Hello,
is the Bitwarden CLI installed in the container?

Thanks in advance :)

**Comment by BlackDex** [comment]
No it isn't. There is also no good reason to have that installed there at all. For one it could cause a security issue because you would have your credentials inside the container. And, also it will not be kept up-to-date with the latest version available from Bitwarden.

The CLI should be installed on the client not on the server.

**Reply by Nollknolle** [reply]
Ok thank you very much!
I would like to write a script that automatically exports my vault. In that case I do it on the client and not on the container directly.
